documentation-site/static/examples/button/with-enchancers.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Button} from 'baseui/button';
+
+function CloudComponent() {
```

Comment:
can we replace this cloud component with an icon component? this code feels like noise that does not contribute to explaining the concept